### ISSUE TITLE
Docker images are not getting deleted from frontend

### ISSUE THREAD

**Issue by sunil3s3** [kwk/docker-registry-frontend#161]
Below is the procedure followed
#docker pull ubuntu
#docker tag ubuntu:latest testserver.xx.com:5000/internal-test/ubuntu:test
#docker push testserver.xx.com:5000/internal-test/ubuntu:test
The image is reflecting in frontend after executing above steps

Now, when image is deleted , still image is visible on frontend
#docker rmi testserver.xx.com:5000/internal-test/ubuntu:test

**Comment by seminarian** [comment]
I could be wrong but the command you give.. Is intended for deletion on your local computer. The 'server-address' is just part of the tag.

**Comment by sunil3s3** [comment]
yes, but when I use docker push the docker image is visible in GUI. The same way if I remove tag it should not be visible in GUI.

Correct me if I'm wrong

**Comment by Nighthawk22** [comment]
@seminarian is right. You just deleted your local image. This will not change the server image.

**Comment by sunil3s3** [comment]
How to delete the server image

**Comment by pavel-veber** [comment]
From **docker-registry-frontend**: https://github.com/kwk/docker-registry-frontend/issues/106

**Comment by pavel-veber** [comment]
From API: https://github.com/docker/docker-registry/issues/988#issuecomment-224280919
